example/rollup/lite.pl

```diff
@@ -1,20 +1,37 @@
 #!/usr/bin/env perl
 use Mojolicious::Lite;
-use FindBin;
-BEGIN { unshift @INC, "$FindBin::Bin/../../lib" }
+use Mojo::File 'curfile';
+require lib;
 
-plugin Webpack =>
-  {process => ['svelte'], dependencies => {core => 'rollup', svelte => [qw(rollup-plugin-svelte svelte)]}};
+$ENV{MOJO_WEBPACK_CONFIG}  = "@{[curfile->sibling('rollup.config.js')]}";
+$ENV{MOJO_WEBPACK_DEBUG}   = 1;
+$ENV{MOJO_WEBPACK_VERBOSE} = 0;
+$ENV{MOJO_WEBPACK_LAZY}    = 0;
+
+my $path = "@{[curfile->dirname->dirname->sibling('lib')]}";
+lib->import($path);
+
+plugin Webpack => {
+  process      => ['svelte'],
+  dependencies => {
+    core   => 'rollup',
+    svelte => [qw(rollup-plugin-svelte svelte)]
+  }
+};
 get '/' => 'index';
-app->start;
+
+{
+  local $ENV{PERL5OPT} = "-I$path";
+  app->start;
+}
 
 __DATA__
 @@ index.html.ep
 <!DOCTYPE html>
 <html>
   <head>
     <title>Mojolicious ♥ Webpack</title>
-    %= asset 'example.css'
+    % # asset 'example.css'
```

Comment:
Is this supposed to be part of this PR?

Reply:
Yeah, but happy to find a solution that either evaluates it, or removes the line completely.
Currently raises an exception:

        Unknown asset name "example.css". at mojolicious-plugin-webpack/lib/Mojolicious/Plugin/Webpack.pm line 80.